app/frontend/src/features/profile/view/Overview.tsx

```diff
@@ -62,9 +61,10 @@ const useStyles = makeStyles((theme) => ({
 
 interface OverviewProps {
   user: User.AsObject;
+  setIsRequesting: Dispatch<SetStateAction<boolean>>;
```

Comment:
`(value: boolean) => void` gives more flexibility imo but if you think being specific is better, leave as is